include/cling/Interpreter/InvocationOptions.h

```diff
@@ -14,16 +14,30 @@
 #include <vector>
 
 namespace cling {
+
+  class CompilerOpts {
+  public:
+    /// \brief Construct CompilerOpts from given arguments, saving them into
+    /// Remaining.
```

Comment:
Yes, it shows up in different font style. More info [here](https://www.stack.nl/~dimitri/doxygen/manual/commands.html#cmdp).